src/parser/schwabJSONEACHistoryParser.ts

```diff
@@ -160,32 +147,45 @@ export function parseEACHistory(input: string): EAC.Transaction[] {
                     awardDate: parseDates(exerciseAndSellRowLine[FIELD_EXERCISE_AND_SELL_AWARD_DATE])[0],
                 }
                 exerciseAndSellRows.push(exerciseAndSellRow);
-                i = i+2;
             }
             eacTransaction.rows = exerciseAndSellRows;
 
-            // Read 1 summary line
-            if (!_.isEqual(parsed.data[i+1], OPTIONS_DETAILS_HEADER)) {
-                throwParsingError(parsed.data[i+1], OPTIONS_DETAILS_HEADER);
+            const optionsDetailsLine = line.OptionExerciseSummary.Details;
+            const optionsDetails = {
+                exerciseCostUSD: parseUSD(optionsDetailsLine.ExerciseCost),
+                grossProceedsUSD: parseUSD(optionsDetailsLine.GrossProceeds),
+                netProceedsUSD: parseUSD(optionsDetailsLine.NetProceeds),
+            }
+            eacTransaction.details = optionsDetails;
+        }
+
+        if (eacTransaction.action === EAC.Action.ExerciseAndHold) {
+            const exerciseAndHoldRows = [];
+            for (let exerciseAndHoldRowLineItem of line.TransactionDetails) {
+                const exerciseAndHoldRowLine = exerciseAndHoldRowLineItem.Details;
+                const exerciseAndHoldRow = {
+                    awardID: exerciseAndHoldRowLine[FIELD_EXERCISE_AND_SELL_AWARD_ID],
+                    sharesExercised: parseQuantity(exerciseAndHoldRowLine[FIELD_EXERCISE_AND_SELL_SHARES_EXERCISED]),
+                    awardPriceUSD: parseUSD(exerciseAndHoldRowLine[FIELD_EXERCISE_AND_SELL_AWARD_PRICE]),
+                    awardType: exerciseAndHoldRowLine[FIELD_EXERCISE_AND_SELL_AWARD_TYPE],
+                    awardDate: parseDates(exerciseAndHoldRowLine[FIELD_EXERCISE_AND_SELL_AWARD_DATE])[0],
```

Comment:
For consistency the exercise and hold transactions should define its own field name constants instead of using the ones for a different transaction type